src/commands/Bunnies.js

```diff
@@ -0,0 +1,48 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+
+class Bunnies extends Command {
+    constructor(...args) {
+        super(...args);
+
+        this.aliases      = ['bunny', 'bunnies'];
+        this.module       = 'Fun';
+        this.description  = 'Random Adorable Bunnies';
+        this.usage        = 'bunny';
+        this.example      = 'bunny';
+        this.cooldown     = 7500;
+        this.expectedArgs = 0;
+    }
+
+    async execute({ message }) {
+		const errorText = `Error: ${this.config.emojis.rabbit || '🐰'} No bunnies found.`;
+        try {

```

Comment:
Inconsistent spacing between errorText and try {

Reply:
fixed